scanner/src/lib/common.ts

```diff
@@ -0,0 +1,10 @@
+import { toast } from "@zerodevx/svelte-toast"
+
+export function consoleLog(message: string, type: string = "info") {
```

Comment:
I'm not sure if toasts are temporary here. If not - maybe we could rename the function then? It caused some confusions while reviewing the code

Reply:
This code is very temporary so it doesn't matter too much.